app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+
```

Comment:
There is no need to add two blank lines between different groups of imports, leave one instead in the future